tests/lametro/test_bills.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from pupa.scrape.bill import Bill
+
+    
+def test_is_restricted(scraper, matter, bill_data_updates):
+    '''
+    Tests that `_is_restricted` returns the correct value, given
+    a dict of matter values.
+    '''
+    field,value,assertion = bill_data_updates
```

Comment:
can i be very particular and request spaces after the commas?

Reply:
ahaha. of course.